keentools_facebuilder/fbloader.py

```diff
@@ -456,42 +456,37 @@ def _unfix_not_in_groups(fb, head):
                     or not cam.auto_focal_estimation)
 
         def _auto_focal_estimation_mode_and_fixes():
-            mode = 'FB_ESTIMATE_VARYING_FOCAL_LENGTH'
             if head.smart_mode():
                 if camera.auto_focal_estimation:
                     if camera.is_in_group():
-                        for cam in head.cameras:
-                            fb.set_focal_length_fixed_at(
-                                cam.get_keyframe(),
-                                cam.image_group != camera.image_group
-                                or not cam.auto_focal_estimation)
-                        mode = 'FB_ESTIMATE_STATIC_FOCAL_LENGTH'
+                        sensor_width = head.sensor_width if head.sensor_width != -1 \
+                            else Config.default_sensor_width
+                        proj_mat = settings.get_projection_matrix()
```

Comment:
I cannot find where settings.get_projection_matrix() method defined.
get_projection_matrix() belongs to camera object, not scene